spec/controllers/nodes_controller_spec.rb

```diff
@@ -116,4 +116,110 @@
       end
     end
   end
+
+  # rubocop:disable RSpec/AnyInstance
+  # rubocop:disable RSpec/ExampleLength
+  # rubocop:disable RSpec/NestedGroups
```

Comment:
Some of these are easily avoidable. For example, NestedGroups can be avoided iirc by using `describe` instead of `context` all the time. I think that this is how the `context` method was meant to be used.